src/Pkg.jl

```diff
@@ -126,7 +126,10 @@ const add = API.add
 """
     Pkg.precompile()
 
-Precompile all the dependencies of the project.
+Precompile all dependencies of the project via a depth-first depdendency tree-based 
+parallel precompilation of all dependencies in the project manifest. 
+Errors will only throw when precompiling the top-level dependencies, given that 
+not all manifest dependencies may be loaded by the top-level dependencies on the given system.
```

Comment:
Perhaps make this a note-admonition?
```suggestion
!!! note
    Errors will only throw when precompiling the top-level dependencies, given that 
    not all manifest dependencies may be loaded by the top-level dependencies on the given system.
```